### PR TITLE
HIVE-26415 : Add epoch time in the information_schema.scheduled_executions view

### PR DESCRIPTION
**Pull request by shreenidhiSaigaonkar**
This commit doesn't contain any secrets.

### What changes were proposed in this pull request?
- It just adds an extra column in the ```scheduled_executions``` view of ```information_schema```

### Why are the changes needed?
- Makes correlation between ```replication_metrics``` and ```scheduled_execution``` easier.

### Does this PR introduce _any_ user-facing change?
- No

### How was this patch tested?
- Manually tested by running the script through beeline interface.

### CODE REVIEW

File: metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql [hunk 1]
```diff
@@ -1980,7 +1980,8 @@ SELECT
   FROM_UNIXTIME(END_TIME) as END_TIME,
   END_TIME-START_TIME as ELAPSED,
   ERROR_MESSAGE,
-  FROM_UNIXTIME(LAST_UPDATE_TIME) AS LAST_UPDATE_TIME
+  FROM_UNIXTIME(LAST_UPDATE_TIME) AS LAST_UPDATE_TIME,
+  START_TIME as START_TIME_SINCE_EPOCH
 FROM
```
Comment (pudidic) [inline metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql:1985]: There is upgrade-4.0.0-alpha1-to-4.0.0-alpha-2.hive.sql, too.

Comment (shreenidhiSaigaonkar) [reply metastore/scripts/upgrade/hive/hive-schema-4.0.0-alpha-2.hive.sql:1985]: @pudidic please check now

**Comment by pudidic** [comment]
+1. Looks good to me.

By the way, I need a write access as a committer. It's been 4 years. I'm new to Apache Hive on GitHub.

**Comment by pudidic** [comment]
I tested the failing TestJdbcWithMiniHS2#testHttpRetryOnServerIdleTimeout on my laptop. It passed. As it's a trivial change, I'll merge it.

**Comment by pudidic** [comment]
LGTM +1. I'll merge it.

**Pull request merged**